frontend/src/constants/logicConstants.js

```diff
@@ -0,0 +1,1141 @@
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.ROI_LARGE]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 2, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+        [RULE_NAMES.WARNINGS]: {
+            enabled_continuous: false,
+            enabled_dichotomous: false,
+            enabled_nested: false,
+            threshold: null, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.HIGH_BMD]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 1, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.HIGH_BMDL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 1, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMD_WARN]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 3, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMDL_WARN]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 3, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.LOW_BMD_FAIL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 10, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+        [RULE_NAMES.LOW_BMDL_FAIL]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 10, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+        [RULE_NAMES.CONTROL_RESIDUAL_HIGH]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: 2, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.CONTROL_STDEV_FIT]: {
+            enabled_continuous: true,
+            enabled_dichotomous: false,
+            enabled_nested: false,
+            threshold: 1.5, // null or float
+            failure_bin: BINS.WARNING,
+        },
+        [RULE_NAMES.DOF_ZERO]: {
+            enabled_continuous: true,
+            enabled_dichotomous: true,
+            enabled_nested: true,
+            threshold: null, // null or float
+            failure_bin: BINS.NO_CHANGE,
+        },
+    },
+};
+
+const headers = [
+    "Test Description",
+    "Continuous",
+    "Dichotomous",
+    "Nested",
+    "Test Threshold",
+    "Bin Placement if Test if Failed",
+    "Notes to Show",
```

Comment:
The "notes to show" can change based on the test-threshold in some cases, and it looks like sometimes the test-threshold is related to this text. for example, if `Ratio of BMD/BMDL (serious)`  test threshold changes to 10, then the text  `BMD/BMDL ratio > 20` should change to `BMD/BMDL ratio > 10`